flink-end-to-end-tests/test-scripts/common.sh

```diff
@@ -101,6 +101,20 @@ function revert_flink_dir() {
     CURL_SSL_ARGS=""
 }
 
+function use_zookeeper_34() {
+  if [ -e "${FLINK_DIR}"/lib/flink-shaded-zookeeper-3.5* ]; then
+    mv "${FLINK_DIR}"/lib/flink-shaded-zookeeper-3.5* "${FLINK_DIR}/opt"
+    mv "${FLINK_DIR}"/opt/flink-shaded-zookeeper-3.4* "${FLINK_DIR}/lib"
+  fi
+}
+
+function use_zookeeper_35() {
+  if [ -e "${FLINK_DIR}"/lib/flink-shaded-zookeeper-3.4* ]; then
+    mv "${FLINK_DIR}"/lib/flink-shaded-zookeeper-3.4* "${FLINK_DIR}/opt"
+    mv "${FLINK_DIR}"/opt/flink-shaded-zookeeper-3.5* "${FLINK_DIR}/lib"
+  fi
+}
```

Comment:
This logic won't work because of `common.sh#backup_flink_dir` which will overwrite `lib` after the end of the test. I think it should work if we only copy `flink-shaded-zookeeper-3.5` instead of moving it.

Reply:
fixed